modules/fedimint-ln-client/src/pay.rs

```diff
@@ -133,17 +138,26 @@ impl LightningPayCreatedOutgoingLnContract {
     }
 
     async fn await_outgoing_contract_funded(
-        common: LightningPayCommon,
+        module_decoder: Decoder,
         global_context: DynGlobalClientContext,
         txid: TransactionId,
-    ) -> Result<(), ()> {
+    ) -> Result<(), GatewayPayError> {
+        let out_point = OutPoint { txid, out_idx: 0 };
         global_context
-            .await_tx_accepted(common.operation_id, txid)
+            .api()
+            .await_output_outcome::<LightningOutputOutcome>(
+                out_point,
+                Duration::MAX,
```

Comment:
Is `Duration::MAX` still a problem for WASM @Maan2003?

Reply:
Yes, Duration::MAX for wasm needs to be `Duration::from_millis(i32::MAX)`
But lets not block this.

Reply:
Fixed, thanks for the pointer